<ui version="4.0" >
 <class>SCRH</class>
 <widget class="QDialog" name="SCRH" >
  <property name="geometry" >
   <rect>
    <x>0</x>
    <y>0</y>
    <width>708</width>
    <height>235</height>
   </rect>
  </property>
  <property name="windowTitle" >
   <string>Dialog</string>
  </property>
  <widget class="QLabel" name="SCRHName" >
   <property name="geometry" >
    <rect>
     <x>160</x>
     <y>10</y>
     <width>531</width>
     <height>71</height>
    </rect>
   </property>
   <property name="font" >
    <font>
     <pointsize>21</pointsize>
     <weight>75</weight>
     <italic>true</italic>
     <bold>true</bold>
     <underline>false</underline>
    </font>
   </property>
   <property name="text" >
    <string>SCRHName</string>
   </property>
  </widget>
  <widget class="QLabel" name="AlbaLogo" >
   <property name="geometry" >
    <rect>
     <x>10</x>
     <y>10</y>
     <width>126</width>
     <height>71</height>
    </rect>
   </property>
   <property name="text" >
    <string/>
   </property>
   <property name="pixmap" >
    <pixmap resource="scrapers.qrc" >:/ALBA_Logo.png</pixmap>
   </property>
  </widget>
  <widget class="QLabel" name="leftJawLabel" >
   <property name="geometry" >
    <rect>
     <x>10</x>
     <y>80</y>
     <width>120</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font" >
    <font>
     <pointsize>25</pointsize>
    </font>
   </property>
   <property name="text" >
    <string>Left Jaw</string>
   </property>
  </widget>
  <widget class="QLabel" name="rightJawLabel" >
   <property name="geometry" >
    <rect>
     <x>560</x>
     <y>80</y>
     <width>142</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font" >
    <font>
     <pointsize>25</pointsize>
    </font>
   </property>
   <property name="text" >
    <string>Right Jaw</string>
   </property>
  </widget>
  <widget class="QGroupBox" name="rightJaw" >
   <property name="geometry" >
    <rect>
     <x>470</x>
     <y>120</y>
     <width>231</width>
     <height>101</height>
    </rect>
   </property>
   <layout class="QGridLayout" >
    <property name="leftMargin" >
     <number>0</number>
    </property>
    <property name="topMargin" >
     <number>0</number>
    </property>
    <property name="rightMargin" >
     <number>0</number>
    </property>
    <property name="bottomMargin" >
     <number>0</number>
    </property>
    <property name="verticalSpacing" >
     <number>2</number>
    </property>
    <item row="0" column="0" >
     <widget class="TauGroupBox" name="rightStepper" >
      <property name="flat" >
       <bool>true</bool>
      </property>
      <property name="showText" >
       <bool>false</bool>
      </property>
      <layout class="QGridLayout" >
       <property name="leftMargin" >
        <number>0</number>
       </property>
       <property name="topMargin" >
        <number>0</number>
       </property>
       <property name="rightMargin" >
        <number>0</number>
       </property>
       <property name="bottomMargin" >
        <number>0</number>
       </property>
       <property name="horizontalSpacing" >
        <number>6</number>
       </property>
       <property name="verticalSpacing" >
        <number>6</number>
       </property>
       <item row="0" column="0" >
        <layout class="QHBoxLayout" >
         <item>
          <widget class="TauValueLabel" name="rightLabelState" >
           <property name="minimumSize" >
            <size>
             <width>50</width>
             <height>22</height>
            </size>
           </property>
           <property name="frameShape" >
            <enum>QFrame::NoFrame</enum>
           </property>
           <property name="frameShadow" >
            <enum>QFrame::Plain</enum>
           </property>
           <property name="showQuality" >
            <bool>false</bool>
           </property>
           <property name="useParentModel" >
            <bool>true</bool>
           </property>
           <property name="model" >
            <string>/State</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="TauStateLed" name="rightLedState" >
           <property name="useParentModel" >
            <bool>true</bool>
           </property>
           <property name="model" >
            <string>/State</string>
           </property>
          </widget>
         </item>
         <item>
          <spacer>
           <property name="orientation" >
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeHint" >
            <size>
             <width>150</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
         <item>
          <widget class="TauLimitSwitch" name="rightLimitN" >
           <property name="useParentModel" >
            <bool>true</bool>
           </property>
           <property name="model" >
            <string>/Limit_switches</string>
           </property>
           <property name="boolIndex" >
            <number>2</number>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="rightLimitLabel" >
           <property name="text" >
            <string>- lim +</string>
           </property>
           <property name="alignment" >
            <set>Qt::AlignCenter</set>
           </property>
          </widget>
         </item>
         <item>
          <widget class="TauLimitSwitch" name="rightLimitP" >
           <property name="useParentModel" >
            <bool>true</bool>
           </property>
           <property name="model" >
            <string>/Limit_switches</string>
           </property>
           <property name="boolIndex" >
            <number>1</number>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item row="1" column="0" >
        <layout class="QHBoxLayout" >
         <item>
          <widget class="QLabel" name="rightStepperLabel" >
           <property name="text" >
            <string>Stepper</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="TauValueLineEdit" name="rightStepperWrite" >
           <property name="useParentModel" >
            <bool>true</bool>
           </property>
           <property name="model" >
            <string>/Position</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="TauValueLabel" name="rightStepperRead" >
           <property name="sizePolicy" >
            <sizepolicy vsizetype="Preferred" hsizetype="Expanding" >
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="useParentModel" >
            <bool>true</bool>
           </property>
           <property name="model" >
            <string>/Position</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="TauConfigLabel" name="rightStepperUnits" >
           <property name="minimumSize" >
            <size>
             <width>35</width>
             <height>24</height>
            </size>
           </property>
           <property name="maximumSize" >
            <size>
             <width>35</width>
             <height>24</height>
            </size>
           </property>
           <property name="useParentModel" >
            <bool>true</bool>
           </property>
           <property name="model" >
            <string>/Position?configuration=unit</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
      </layout>
     </widget>
    </item>
    <item row="1" column="0" >
     <widget class="TauGroupBox" name="rightEnc" >
      <property name="flat" >
       <bool>true</bool>
      </property>
      <property name="showText" >
       <bool>false</bool>
      </property>
      <layout class="QGridLayout" >
       <property name="leftMargin" >
        <number>0</number>
       </property>
       <property name="topMargin" >
        <number>0</number>
       </property>
       <property name="rightMargin" >
        <number>0</number>
       </property>
       <property name="bottomMargin" >
        <number>0</number>
       </property>
       <property name="horizontalSpacing" >
        <number>6</number>
       </property>
       <property name="verticalSpacing" >
        <number>6</number>
       </property>
       <item row="0" column="0" >
        <layout class="QHBoxLayout" >
         <item>
          <widget class="QPushButton" name="rightAbort" >
           <property name="maximumSize" >
            <size>
             <width>60</width>
             <height>25</height>
            </size>
           </property>
           <property name="text" >
            <string>Abort</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="rightEncLabel" >
           <property name="text" >
            <string>Encoder</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="TauValueLabel" name="rightEncRead" >
           <property name="sizePolicy" >
            <sizepolicy vsizetype="Preferred" hsizetype="Expanding" >
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="useParentModel" >
            <bool>true</bool>
           </property>
           <property name="model" >
            <string>/Position</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="TauConfigLabel" name="rightEncUnits" >
           <property name="minimumSize" >
            <size>
             <width>35</width>
             <height>24</height>
            </size>
           </property>
           <property name="maximumSize" >
            <size>
             <width>35</width>
             <height>24</height>
            </size>
           </property>
           <property name="useParentModel" >
            <bool>true</bool>
           </property>
           <property name="model" >
            <string>/Position?configuration=unit</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
      </layout>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QGroupBox" name="leftJaw" >
   <property name="geometry" >
    <rect>
     <x>10</x>
     <y>120</y>
     <width>231</width>
     <height>101</height>
    </rect>
   </property>
   <layout class="QGridLayout" >
    <property name="leftMargin" >
     <number>0</number>
    </property>
    <property name="topMargin" >
     <number>0</number>
    </property>
    <property name="rightMargin" >
     <number>0</number>
    </property>
    <property name="bottomMargin" >
     <number>0</number>
    </property>
    <property name="verticalSpacing" >
     <number>2</number>
    </property>
    <item row="0" column="0" >
     <widget class="TauGroupBox" name="leftStepper" >
      <property name="flat" >
       <bool>true</bool>
      </property>
      <property name="showText" >
       <bool>false</bool>
      </property>
      <layout class="QGridLayout" >
       <property name="leftMargin" >
        <number>0</number>
       </property>
       <property name="topMargin" >
        <number>0</number>
       </property>
       <property name="rightMargin" >
        <number>0</number>
       </property>
       <property name="bottomMargin" >
        <number>0</number>
       </property>
       <property name="horizontalSpacing" >
        <number>6</number>
       </property>
       <property name="verticalSpacing" >
        <number>6</number>
       </property>
       <item row="0" column="0" >
        <layout class="QHBoxLayout" >
         <item>
          <widget class="TauValueLabel" name="leftLabelState" >
           <property name="minimumSize" >
            <size>
             <width>50</width>
             <height>22</height>
            </size>
           </property>
           <property name="frameShape" >
            <enum>QFrame::NoFrame</enum>
           </property>
           <property name="frameShadow" >
            <enum>QFrame::Plain</enum>
           </property>
           <property name="showQuality" >
            <bool>false</bool>
           </property>
           <property name="useParentModel" >
            <bool>true</bool>
           </property>
           <property name="model" >
            <string>/State</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="TauStateLed" name="leftLedState" >
           <property name="useParentModel" >
            <bool>true</bool>
           </property>
           <property name="model" >
            <string>/State</string>
           </property>
          </widget>
         </item>
         <item>
          <spacer>
           <property name="orientation" >
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeHint" >
            <size>
             <width>150</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
         <item>
          <widget class="TauLimitSwitch" name="leftLimitN" >
           <property name="useParentModel" >
            <bool>true</bool>
           </property>
           <property name="model" >
            <string>/Limit_switches</string>
           </property>
           <property name="boolIndex" >
            <number>2</number>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="leftLimitLabel" >
           <property name="text" >
            <string>- lim +</string>
           </property>
           <property name="alignment" >
            <set>Qt::AlignCenter</set>
           </property>
          </widget>
         </item>
         <item>
          <widget class="TauLimitSwitch" name="leftLimitP" >
           <property name="useParentModel" >
            <bool>true</bool>
           </property>
           <property name="model" >
            <string>/Limit_switches</string>
           </property>
           <property name="boolIndex" >
            <number>1</number>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item row="1" column="0" >
        <layout class="QHBoxLayout" >
         <item>
          <widget class="QLabel" name="leftStepperLabel" >
           <property name="text" >
            <string>Stepper</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="TauValueLineEdit" name="leftStepperWrite" >
           <property name="useParentModel" >
            <bool>true</bool>
           </property>
           <property name="model" >
            <string>/Position</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="TauValueLabel" name="leftStepperRead" >
           <property name="sizePolicy" >
            <sizepolicy vsizetype="Preferred" hsizetype="Expanding" >
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="useParentModel" >
            <bool>true</bool>
           </property>
           <property name="model" >
            <string>/Position</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="TauConfigLabel" name="leftStepperUnits" >
           <property name="minimumSize" >
            <size>
             <width>35</width>
             <height>24</height>
            </size>
           </property>
           <property name="maximumSize" >
            <size>
             <width>35</width>
             <height>24</height>
            </size>
           </property>
           <property name="useParentModel" >
            <bool>true</bool>
           </property>
           <property name="model" >
            <string>/Position?configuration=unit</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
      </layout>
     </widget>
    </item>
    <item row="1" column="0" >
     <widget class="TauGroupBox" name="leftEnc" >
      <property name="flat" >
       <bool>true</bool>
      </property>
      <property name="showText" >
       <bool>false</bool>
      </property>
      <layout class="QGridLayout" >
       <property name="leftMargin" >
        <number>0</number>
       </property>
       <property name="topMargin" >
        <number>0</number>
       </property>
       <property name="rightMargin" >
        <number>0</number>
       </property>
       <property name="bottomMargin" >
        <number>0</number>
       </property>
       <property name="horizontalSpacing" >
        <number>6</number>
       </property>
       <property name="verticalSpacing" >
        <number>6</number>
       </property>
       <item row="0" column="0" >
        <layout class="QHBoxLayout" >
         <item>
          <widget class="QPushButton" name="leftAbort" >
           <property name="maximumSize" >
            <size>
             <width>60</width>
             <height>25</height>
            </size>
           </property>
           <property name="text" >
            <string>Abort</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="leftEncLabel" >
           <property name="text" >
            <string>Encoder</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="TauValueLabel" name="leftEncRead" >
           <property name="sizePolicy" >
            <sizepolicy vsizetype="Preferred" hsizetype="Expanding" >
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="useParentModel" >
            <bool>true</bool>
           </property>
           <property name="model" >
            <string>/Position</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="TauConfigLabel" name="leftEncUnits" >
           <property name="minimumSize" >
            <size>
             <width>35</width>
             <height>24</height>
            </size>
           </property>
           <property name="maximumSize" >
            <size>
             <width>35</width>
             <height>24</height>
            </size>
           </property>
           <property name="useParentModel" >
            <bool>true</bool>
           </property>
           <property name="model" >
            <string>/Position?configuration=unit</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
      </layout>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QGroupBox" name="gapAndOffset" >
   <property name="geometry" >
    <rect>
     <x>250</x>
     <y>180</y>
     <width>211</width>
     <height>41</height>
    </rect>
   </property>
   <property name="flat" >
    <bool>true</bool>
   </property>
   <layout class="QHBoxLayout" >
    <property name="spacing" >
     <number>0</number>
    </property>
    <property name="leftMargin" >
     <number>0</number>
    </property>
    <property name="topMargin" >
     <number>0</number>
    </property>
    <property name="rightMargin" >
     <number>0</number>
    </property>
    <property name="bottomMargin" >
     <number>0</number>
    </property>
    <item>
     <widget class="TauGroupBox" name="gap" >
      <property name="flat" >
       <bool>true</bool>
      </property>
      <property name="showText" >
       <bool>false</bool>
      </property>
      <layout class="QGridLayout" >
       <property name="leftMargin" >
        <number>0</number>
       </property>
       <property name="topMargin" >
        <number>0</number>
       </property>
       <property name="rightMargin" >
        <number>0</number>
       </property>
       <property name="bottomMargin" >
        <number>0</number>
       </property>
       <item row="0" column="0" >
        <layout class="QHBoxLayout" >
         <item>
          <widget class="TauValueLabel" name="gapRead" >
           <property name="sizePolicy" >
            <sizepolicy vsizetype="Preferred" hsizetype="Expanding" >
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="useParentModel" >
            <bool>true</bool>
           </property>
           <property name="model" >
            <string>/Position</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="TauConfigLabel" name="gapUnits" >
           <property name="minimumSize" >
            <size>
             <width>35</width>
             <height>24</height>
            </size>
           </property>
           <property name="maximumSize" >
            <size>
             <width>35</width>
             <height>24</height>
            </size>
           </property>
           <property name="useParentModel" >
            <bool>true</bool>
           </property>
           <property name="model" >
            <string>/Position?configuration=unit</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
      </layout>
     </widget>
    </item>
    <item>
     <widget class="TauGroupBox" name="offset" >
      <property name="flat" >
       <bool>true</bool>
      </property>
      <property name="showText" >
       <bool>false</bool>
      </property>
      <layout class="QGridLayout" >
       <property name="leftMargin" >
        <number>0</number>
       </property>
       <property name="topMargin" >
        <number>0</number>
       </property>
       <property name="rightMargin" >
        <number>0</number>
       </property>
       <property name="bottomMargin" >
        <number>0</number>
       </property>
       <item row="0" column="0" >
        <layout class="QHBoxLayout" >
         <item>
          <widget class="TauValueLabel" name="offsetRead" >
           <property name="sizePolicy" >
            <sizepolicy vsizetype="Preferred" hsizetype="Expanding" >
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="useParentModel" >
            <bool>true</bool>
           </property>
           <property name="model" >
            <string>/Position</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="TauConfigLabel" name="offsetUnits" >
           <property name="minimumSize" >
            <size>
             <width>35</width>
             <height>24</height>
            </size>
           </property>
           <property name="maximumSize" >
            <size>
             <width>35</width>
             <height>24</height>
            </size>
           </property>
           <property name="useParentModel" >
            <bool>true</bool>
           </property>
           <property name="model" >
            <string>/Position?configuration=unit</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
      </layout>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QLabel" name="gapLabel" >
   <property name="geometry" >
    <rect>
     <x>250</x>
     <y>140</y>
     <width>88</width>
     <height>40</height>
    </rect>
   </property>
   <property name="font" >
    <font>
     <pointsize>25</pointsize>
    </font>
   </property>
   <property name="text" >
    <string>Gap</string>
   </property>
  </widget>
  <widget class="QLabel" name="offsetLabel" >
   <property name="geometry" >
    <rect>
     <x>350</x>
     <y>140</y>
     <width>88</width>
     <height>40</height>
    </rect>
   </property>
   <property name="font" >
    <font>
     <pointsize>25</pointsize>
    </font>
   </property>
   <property name="text" >
    <string>Offset</string>
   </property>
  </widget>
 </widget>
 <customwidgets>
  <customwidget>
   <class>TauGroupBox</class>
   <extends>QGroupBox</extends>
   <header>tau.widget</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>TauStateLed</class>
   <extends>QLabel</extends>
   <header>tau.widget</header>
  </customwidget>
  <customwidget>
   <class>TauLimitSwitch</class>
   <extends>QLabel</extends>
   <header>tau.widget</header>
  </customwidget>
  <customwidget>
   <class>TauValueLineEdit</class>
   <extends>QLineEdit</extends>
   <header>tau.widget</header>
  </customwidget>
  <customwidget>
   <class>TauConfigLabel</class>
   <extends>QLabel</extends>
   <header>tau.widget</header>
  </customwidget>
  <customwidget>
   <class>TauValueLabel</class>
   <extends>QLabel</extends>
   <header>tau.widget</header>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="scrapers.qrc" />
 </resources>
 <connections/>
</ui>
